src/theme/components/styledList.ts

```diff
@@ -0,0 +1,23 @@
+import { baseListStyles, baseUnorderedStyles } from "./list";
+
+const StyledList = {
+  baseStyle: ({ textSize = "default" }) => ({
+    fontSize: `text.${textSize}`,
+    ...baseListStyles(false, true),
+    ...baseUnorderedStyles(true),
+  }),
+  variants: {
+    capped: {
+      li: {
+        borderLeft: "2px solid",
+        borderRight: "2px solid",
```

Comment:
Sorry, it wasn't clearly apparent in the Figma designs, but the styling for `desktop` and `mobile` are slightly different.  For `desktop`, the right border and padding should be removed.  The border on both ends is only needed for the `mobile` view.  

For a better example, see the link below: 
https://www.figma.com/file/CRPxcR5Olmz50IrFZxvLKE/DRB---Search-Highlighting-(SFR-1519)?node-id=1279%3A12441

Reply:
Right. This could use it's own figma card like the other components.

Reply:
Yes, the designs in the Visual Design Library (the Figma Main file) are coming.